templates/index.html

```diff
@@ -7,7 +7,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>{{title}}</title>
-    <link rel="canonical" href="https://www.srihash.org"/>
+    <link rel="canonical" href="https://www.srihash.org/">
```

Comment:
BTW the idea was to actually do this dynamically and not hardcoded like now. Not sure how to get the value from Hapi though.

Reply:
Dynamically, why? I thought the idea was to say that one domain should be preferred over the other and that's it.

Reply:
Dynamically meaning, get the current domain Hapi is using and not hardcoding it like this.

Reply:
But anyway, this isn't something to deal with on this PR.